.github/workflows/ci.yml

```diff
@@ -10,7 +10,11 @@ jobs:
 
     strategy:
       matrix:
-        elixir-otp: [{otp: 24.x, elixir: 1.12.x}, {otp: 25.x, elixir: 1.13.x}]
+        elixir-otp:
+          - otp: 25.x
+            elixir: 1.14.x
+          - otp: 26.x
+            elixir: 1.15.x
```

Comment:
why did you remove 1.12 and 1.13?

Reply:
Because they are not backwards compatible with the changed dependencies.

Reply:
This will likely require a major version bump to signify this change.

Reply:
I don't want to drop support for older versions of Elixir just for a dev dependency that I have never even used since taking over the project.

you can probably add `--only test` to mix deps.get in CI to avoid having to update benchee.

Reply:
I am down to drop benchee as a dependency altogether. However there are other changes around `ExUnit.Server.modules_loaded()` that are not backwards compatible.

Reply:
that is a private elixir API, and is in a unit test.

I would just add a conditional that checks if the function is exported and call the one that is

`function_exported?(ExUnit.Server, :modules_loaded, 0)`, etc